src/python/pants_release/generate_github_workflows.py

```diff
@@ -932,8 +932,13 @@ def build_wheels_job(
                             "if": "needs.release_info.outputs.is-release == 'true'",
                             "run": dedent(
                                 """\
+                                PANTS_VER=$(PEX_INTERPRETER=1 dist/src.python.pants/pants-pex.pex -c "import pants.version;print(pants.version.VERSION)")
+                                PY_VER=$(PEX_INTERPRETER=1 dist/src.python.pants/pants-pex.pex -c "import sys;print(f'cp{sys.version_info[0]}{sys.version_info[1]}')")
+                                PLAT=$(PEX_INTERPRETER=1 dist/src.python.pants/pants-pex.pex -c "import os;print(f'{os.uname().sysname.lower()}_{os.uname().machine.lower()}')")
                                 PEX_FILENAME=pants.$PANTS_VER-$PY_VER-$PLAT.pex
+
                                 mv dist/src.python.pants/pants-pex.pex dist/src.python.pants/$PEX_FILENAME
+                                echo "PEX_FILENAME=$PEX_FILENAME" >> "$GITHUB_ENV"
```

Comment:
Totally minor (feel free to ignore), but I [suggest](https://huonw.github.io/blog/2025/02/ci-tee/) using `tee -a` to get "automatic" introspection/debugging.

```suggestion
                                echo "PEX_FILENAME=$PEX_FILENAME" | tee -a "$GITHUB_ENV"
```

Reply:
Great idea!